jdaviz/configs/imviz/helper.py

```diff
@@ -340,6 +345,22 @@ def layer_is_table_data(layer):
     return isinstance(layer, BaseData) and layer.ndim == 1
 
 
+def get_bottom_layer(viewer):
+    """
+    Get the first-loaded image layer in Imviz.
+    """
+    image_layers = [lyr.layer for lyr in viewer.layers
+                    if lyr.visible and layer_is_image_data(lyr.layer)]
+    if not len(image_layers):
+        return image_layers
```

Comment:
so this would return an empty list?  Does that make more sense than `None` or raising an error?  (If its not public, it probably doesn't matter too much)